operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/BulkDependentResourceReconciler.java

```diff
@@ -24,18 +24,24 @@ class BulkDependentResourceReconciler<R, P extends HasMetadata>
 
   @Override
   public ReconcileResult<R> reconcile(P primary, Context<P> context) {
-    final var desiredResources = bulkDependentResource.desiredResources(primary, context);
+
     Map<String, R> actualResources = bulkDependentResource.getSecondaryResources(primary, context);
+    if (!(bulkDependentResource instanceof Creator<?, ?>)
+        && !(bulkDependentResource instanceof Deleter<?>)
+        && !(bulkDependentResource instanceof Updater<?, ?>)) {
+      return ReconcileResult
+          .aggregatedResult(actualResources.values().stream().map(ReconcileResult::noOperation)
+              .toList());
+    }
+
+    final var desiredResources = bulkDependentResource.desiredResources(primary, context);
 
     // remove existing resources that are not needed anymore according to the primary state
     deleteExtraResources(desiredResources.keySet(), actualResources, primary, context);
```

Comment:
TBH this is interesting we should probably check if we are deleter? But what if not ? :)

Reply:
Not sure I understand…

Reply:
I updated it, before deleting extra resources was called all the time now we check if this DR is a `deleter`.